ePanchayat.API/Repository/AnnouncementRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using ePanchayat.API.Interfaces;
+using ePanchayat.API.Models;
+using System.Data;
+
+namespace ePanchayat.API.Repository
+{
+    public class AnnouncementRepository : IAnnouncementRepository
+    {
+        private readonly ISqlDataAccess _sqlDataAccess;
+
+        public AnnouncementRepository(ISqlDataAccess sqlDataAccess)
+        {
+            _sqlDataAccess = sqlDataAccess;
+        }
+
+        public List<Announcement> GetAll()
+        {
+            var dataset = _sqlDataAccess.GetDataSetByStoredProc("AnnouncementGet_sp");
+            return Map(dataset);
+        }
+
+        public Announcement GetById(int AnnouncementId)
+        {
+            var parameters = new Dictionary<string, object>
+            {
+                { "@AnnouncementId", AnnouncementId }
+            };
+            var dataset = _sqlDataAccess.GetDataSetByStoredProc("AnnouncementGet_sp", parameters);
+
+            return Map(dataset).First();
+        }
+
+        public bool Remove(Announcement Announcement)
+        {

```

Comment:
As you are proceeding with Get implementation for now, let we add task for Save/Update/Delete operations for better tracking